pom.xml

```diff
@@ -1047,8 +1046,9 @@
     </dependency>
 
     <dependency>
-      <groupId>org.apache.drill</groupId>
-      <artifactId>drill-shaded-guava</artifactId>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>${guava.version}</version>
```

Comment:
Please remove the `version` line, version should be obtained from the dependencies management block.